tests/sentence_transformers/test_sentence_transformers_model_export.py

```diff
@@ -324,6 +325,45 @@ def test_model_pyfunc_save_load(basic_model, model_path):
     np.testing.assert_array_equal(emb1, emb3)
 
 
+def test_model_pyfunc_predict_with_params(basic_model, tmp_path):
+    sentence = "hello world and hello mlflow"
+    params = {"batch_size": 16}
+
+    model_path = tmp_path / "model1"
+    signature = infer_signature(sentence, params=params)
+    mlflow.sentence_transformers.save_model(basic_model, model_path, signature=signature)
+    loaded_pyfunc = pyfunc.load_model(model_uri=model_path)
+    embedding_dim = basic_model.get_sentence_embedding_dimension()
+
+    emb0 = loaded_pyfunc.predict(sentence, params)
+    assert emb0.shape == (1, embedding_dim)
+
+    with pytest.raises(MlflowException, match=r"Invalid parameters found"):
+        loaded_pyfunc.predict(sentence, {"batch_size": "16"})
```

Comment:
Does the error message contain how the parameters are invalid?

Reply:
Yes